src/ui-kit/components/sidenav/sidenav/sidenav.template.html

```diff
@@ -30,6 +30,7 @@
       [ngClass]="{
         'active': isSelected($index),
         'completed': child.iconClass == 'completed',
+        'warning': child.iconClass == 'warning',
```

Comment:
Does this CSS already exist or is this custom CSS?

Reply:
its in the sam-front-end css, don't see it in sam-ui-elements anywhere

Reply:
Is that where the rest of these classes are as well?

Reply:
yep im seeing it the sam-front-end css as well, should those get added into sam-ui-elements css?

Reply:
Yeah that is a bigger deal.